app/assets/stylesheets/custom.css.scss

```diff
@@ -12,8 +12,8 @@ header div h2 {
 }
 
 .overlay {
-  background:transparent; 
-  position:relative; 
+  background:transparent;
+  position:relative;
```

Comment:
Properties should be ordered background, height, margin-top, position, top, width<br>Colon after property should be followed by one space